TsFormTest::should_generate_FormModels fails on Windows

TsFormTest::should_generate_FormModels has been failing on Windows since this change https://github.com/vaadin/flow/commit/334ac218e7c5 has been made:

```
java.lang.AssertionError: Found differences in generated file: L14 :: expected: [ * @see {@link file:///.../fusion-endpoint/src/test/java/com/vaadin/flow/server/connect/generator/tsmodel/TsFormEndpoint.java}] got [ * @see {@link file://C:\BuildAgent\work\ea6c1ff9b5606009\fusion-endpoint\src\test\java\com\vaadin\flow\server\connect\generator\tsmodel\TsFormEndpoint.java}]
	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at com.vaadin.flow.server.connect.generator.tsmodel.TsFormTest.should_generate_FormModels(TsFormTest.java:117)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:54)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
```

https://bender.vaadin.com/viewLog.html?buildId=221924&tab=buildResultsDiv&buildTypeId=Flow_Flow70_Nightly_ValidateFlowMasterWithWindows

--- a/fusion-endpoint/src/test/java/com/vaadin/flow/server/connect/generator/tsmodel/TsFormTest.java
+++ b/fusion-endpoint/src/test/java/com/vaadin/flow/server/connect/generator/tsmodel/TsFormTest.java
@@ -23,6 +23,7 @@ import java.util.Collections;
 import java.util.List;
 import java.util.stream.Collectors;
 
+import org.junit.Assert;
 import org.junit.Test;
 
 import com.vaadin.flow.server.connect.generator.endpoints.AbstractEndpointGeneratorBaseTest;
@@ -116,11 +117,14 @@ public class TsFormTest extends AbstractEndpointGeneratorBaseTest {
         int line = 0;
         List<String> faultyLines = new ArrayList<>();
         for (String expectedLine : expected) {
+            String actualLine = content.get(line);
             // ignore the line for file reference, as the file path syntax is different on Windows
-            if (!expectedLine.contains("@see {@link file:///...") &&
-                !expectedLine.equals(content.get(line))) {
+            if (isFileRefenreceLine(expectedLine)) {
+                Assert.assertTrue("should have the file reference", 
+                        isFileRefenreceLine(actualLine)); 
+            } else if (!expectedLine.equals(actualLine)) {
                 faultyLines.add(String.format("L%d :: expected: [%s] got [%s]",
-                        line + 1, expectedLine, content.get(line)));
+                        line + 1, expectedLine, actualLine));
             }
             line++;
         }
@@ -128,5 +132,9 @@ public class TsFormTest extends AbstractEndpointGeneratorBaseTest {
                 "Found differences in generated file: " + faultyLines.stream()
                         .collect(Collectors.joining("\n")),
                 faultyLines.isEmpty());
+    }
+
+    private boolean isFileRefenreceLine(String line) {
+        return line.contains("@see {@link file:");
     }
 }

--- a/fusion-endpoint/src/test/java/com/vaadin/flow/server/connect/generator/tsmodel/TsFormTest.java
+++ b/fusion-endpoint/src/test/java/com/vaadin/flow/server/connect/generator/tsmodel/TsFormTest.java
@@ -116,7 +116,9 @@ public class TsFormTest extends AbstractEndpointGeneratorBaseTest {
         int line = 0;
         List<String> faultyLines = new ArrayList<>();
         for (String expectedLine : expected) {
-            if (!expectedLine.equals(content.get(line))) {
+            // ignore the line for file reference, as the file path syntax is different on Windows
+            if (!expectedLine.contains("@see {@link file:///...") &&
+                !expectedLine.equals(content.get(line))) {
                 faultyLines.add(String.format("L%d :: expected: [%s] got [%s]",
                         line + 1, expectedLine, content.get(line)));
             }

--- a/fusion-endpoint/src/test/java/com/vaadin/flow/server/startup/fusion/DevModeInitializerEndpointTest.java
+++ b/fusion-endpoint/src/test/java/com/vaadin/flow/server/startup/fusion/DevModeInitializerEndpointTest.java
@@ -167,7 +167,6 @@ public class DevModeInitializerEndpointTest {
         DevModeInitializer devModeInitializer = new DevModeInitializer();
         devModeInitializer.onStartup(classes, servletContext);
         waitForDevModeServer();
-        Thread.sleep(200);
         Assert.assertTrue("Should generate OpenAPI spec if Endpoint is used.",
                 generatedOpenApiJson.exists());
     }

--- a/fusion-endpoint/src/test/java/com/vaadin/flow/server/startup/fusion/DevModeInitializerEndpointTest.java
+++ b/fusion-endpoint/src/test/java/com/vaadin/flow/server/startup/fusion/DevModeInitializerEndpointTest.java
@@ -165,7 +165,7 @@ public class DevModeInitializerEndpointTest {
 
         Assert.assertFalse(generatedOpenApiJson.exists());
         DevModeInitializer devModeInitializer = new DevModeInitializer();
-        devModeInitializer.process(classes, servletContext);
+        devModeInitializer.onStartup(classes, servletContext);
         waitForDevModeServer();
         Thread.sleep(200);
         Assert.assertTrue("Should generate OpenAPI spec if Endpoint is used.",
@@ -180,7 +180,7 @@ public class DevModeInitializerEndpointTest {
                 .toFile();
 
         Assert.assertFalse(generatedOpenApiJson.exists());
-        devModeInitializer.process(classes, servletContext);
+        devModeInitializer.onStartup(classes, servletContext);
         waitForDevModeServer();
         Assert.assertFalse(
                 "Should not generate OpenAPI spec if Endpoint is not used.",
@@ -208,7 +208,7 @@ public class DevModeInitializerEndpointTest {
 
         assertFalse(ts1.exists());
         assertFalse(ts2.exists());
-        devModeInitializer.process(classes, servletContext);
+        devModeInitializer.onStartup(classes, servletContext);
         waitForDevModeServer();
         assertTrue(ts1.exists());
         assertTrue(ts2.exists());

